pages/policies/new.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from "react";
+import Input from "components/Input";
+import TextArea from "components/TextArea";
+import Button from "components/Button";
+import styles from "styles/modules/Policy.module.scss";
+import { useTheme } from "providers/theme";
+import { useRouter } from "next/router";
+import ExternalLink from "components/ExternalLink";
+import { schema } from "schemas/new-policy-form";
+import { useFormValidation } from "hooks/useFormValidation";
+
+const NewPolicy = () => {
+  const { theme } = useTheme();
+  // const [validationResult, setValidationResult] = useState(null);
```

Comment:
still need this?

Reply:
No, this was in preparation for validating the rego code but that endpoint is done yet so I can remove.